src/index.js

```diff
@@ -1,3 +1,7 @@
+for( i = 1; i <= 20; i ++ ){
+    document.write( i + "<br>");
+
+
```

Comment:
Tu entendimiento del bucle for va bien! solo te hace falta enfocarlo a resolver la funcionalidad del cipher.